providers/azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/compute/config/AzureNameValidator.java

```diff
@@ -54,7 +55,7 @@ private CharMatcher getAcceptableRange() {
 
    protected IllegalArgumentException exception(String name, String reason) {
       return new IllegalArgumentException(
-            String.format("Object '%s' doesn't match Azure naming constraints. " + "Reason: %s.", name,
+            String.format("Object '%s' doesn't match Azure naming constraints. " + "Reason: %s", name,
```

Comment:
Same here. There is no need to break the string.

Reply:
:up: with no period